jerry-core/parser/js/js-parser-tagged-template-literal.c

```diff
@@ -129,7 +132,6 @@ parser_tagged_template_literal_freeze_array (ecma_object_t *obj_p)
   ecma_op_ordinary_object_prevent_extensions (obj_p);
   ecma_extended_object_t *ext_obj_p = (ecma_extended_object_t *) obj_p;
   ext_obj_p->u.array.length_prop_and_hole_count &= (uint32_t) ~ECMA_PROPERTY_FLAG_WRITABLE;
```

Comment:
Delete this line as well.

Reply:
After the rewrite I cannot delete it from here, since `ECMA_PROPERTY_FLAG_WRITABLE` flags should be kept until the array if frozen. If we remove it at the beginning we cannot add properties to the array. I'd keep it here.

Reply:
Optionally we could use the same technique as above: set the correct value directly. It is faster than flipping a flag (read/write). It is up to you.